organizations/serializers.py

```diff
@@ -10,9 +13,32 @@
 # pylint: disable=too-few-public-methods
 class OrganizationSerializer(serializers.ModelSerializer):
     """ Serializes the Organization object."""
+    logo_url = serializers.CharField(write_only=True, required=False)
+
     class Meta(object):  # pylint: disable=missing-docstring
         model = models.Organization
-        fields = '__all__'
+        fields = ('id', 'created', 'modified', 'name', 'short_name', 'description', 'logo',
+                  'active', 'logo_url',)
+
+    def update_logo(self, obj, logo_url):  # pylint: disable=missing-docstring
+        logo = requests.get(logo_url)
+        obj.logo.save(logo_url.split('/')[-1], ContentFile(logo.content))
```

Comment:
`split('/')[-1]` might not be sufficient for all URLs. Like `?`, `/`, or `#` can be at end. I think `urllib` should be able to hook you up with a way to get the last component?